.gitignore

```diff
@@ -15,6 +15,11 @@
 /node_modules
```

Comment:
@yannham answering here to be able to toggle conversations

Issue https://github.com/stephank/yarn-plugin-nixify/issues/52 has 1 answer and I find it unsatisfying: "Use this even less popular tool which supports your use case"

I thought the author would answer at some point with a solution, but it's been nearly 2 weeks now, it may be time to call the time of death of this PR.

I created https://github.com/tweag/nickel/issues/974 to track the need underlying this PR (and which also answers your question in the Dependabot PR) and suggesting some alternatives.